website/docs/components/badge/index.md

```diff
@@ -17,6 +17,8 @@ navigation:
     - pill
     - tag
     - label
+status:
+  updated: 4.17.0
```

Comment:
I updated this manually. Is it ok to leave or should I revert and let the build tool add it as part of the next branch release?

Reply:
I would let the automation handle it for you.

Reply:
Thanks, I will leave this one alone to avoid triggering a re-review.